packages/travix-ui-kit/components/slidingPanel/slidingPanel.js

```diff
@@ -132,6 +133,62 @@ export default class SlidingPanel extends Component {
     const panelClassName = classnames(getClassNamesWithMods('ui-sliding-panel', panelMods), className);
     const overlayClassName = getClassNamesWithMods('ui-sliding-panel-overlay', overlayMods);
 
+    let titleContent = title ? (
```

Comment:
After the migration this will going to die since title property will be deprecated.